Library/Formula/cyassl.rb

```diff
@@ -55,6 +67,8 @@ def install
       args << "--disable-fastmath" << "--disable-fasthugemath"
     end
 
+    args << "--enable-aesni" if Hardware::CPU.aes?
```

Comment:
Sure, Will fix now.

Does that disable the option if people build from source?